### PR TITLE
Test: add test for health/check

### PR DESCRIPTION
**Pull request by HuKeping**
Signed-off-by: Hu Keping <hukeping@huawei.com>

### CODE REVIEW

**Comment by HuKeping** [comment]
This test include checking for both file exists and not exists

File: health/checks/checks_test.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	"testing"
+)
+
+func TestFileChecker(t *testing.T) {
+	if checker := FileChecker("/tmp"); checker.Check() == nil {
+		t.Errorf("/tmp was expected as exists")
+	}
+
+	if checker := FileChecker("NoSuchFileFromMoon"); checker.Check() != nil {
+		t.Errorf("NoSuchFileFromMoon was expected as not exists, error:%v", checker.Check())
```
Comment (stevvooe) [inline health/checks/checks_test.go]: Save the error above, rather than running the check a second time.

Comment (HuKeping) [inline health/checks/checks_test.go]: sure, I'll update

**Comment by HuKeping** [comment]
@stevvooe  updated, PTAL

**Pull request merged**